composer.json

```diff
@@ -16,16 +16,33 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.9",
+        "symfony/console": "~3.3|~4.0",
+        "symfony/config": "~3.3|~4.0",
+        "symfony/dependency-injection": "~3.3|~4.0",
+        "symfony/framework-bundle": "~3.3|~4.0",
```

Comment:
technically yes. But looking at others (twigbundle, securitybundle) they all have framework-bundle in require-dev.

A hard dep on frameworkbundle, solely for ContainerAwareCommand seems like a bad ideaa.. hence it should extend Command ASAP :)